json-definitions/v1/test-result/index.json

```diff
@@ -6,22 +6,49 @@
       "type": "string"
     },
     "testStationName": {
-      "type": "string"
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 999
     },
     "testStationPNumber": {
-      "type": "string"
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 20
     },
     "testStationType": {
-      "type": "string"
+      "anyOf": [
+        {
+          "$ref": "../enums/testStationType.enum.json"
+        },
+        {
+          "type": "null"
+        }
+      ]
     },
     "testerName": {
-      "type": "string"
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 60
     },
     "testerStaffId": {
-      "type": "string"
+      "type": [
+        "string",
+        "null"
```

Comment:
Just comparing this with the testResultsCommonSchema below:

`testerStaffId: Joi.string().max(36).required().allow(''),` 

It doesn't look like `null` is allowable, even though empty string is, so it worth updating to this instead?

```
"testerStaffId": {
  "type": "string",
  "maxLength": 36,
  "minLength": 0
}
```

Reply:
I'll leave the min length off but will fix this, that's just a small error with my reading abilities